### PR TITLE
📝 Scribe: Add tests for UpdateMeetingRequest

### PR DESCRIPTION
**Pull request by GarethClarridge**
🎯 **Coverage:** `App\Http\Requests\UpdateMeetingRequest` is now thoroughly tested.

📋 **Tests added:**
- `authorize_allows_admin_with_permission`
- `authorize_denies_guests`
- `authorize_denies_user_without_permission`
- `validation_rules` (Data Provider with multiple scenarios)
- `slug_must_be_unique`
- `slug_can_be_same_as_current_meeting`
- `page_id_must_exist`
- `page_id_must_be_unique_except_current`

🔍 **Why:** `UpdateMeetingRequest` handles critical data integrity for church meetings and was previously only minimally tested in a generic authorization test. Consolidating tests into a dedicated unit test file improves maintainability and ensures all validation edge cases are covered.

✅ **Results:** All tests pass, PHPStan clean, Pint formatted.

---
*PR created automatically by Jules for task [7451371237623771014](https://jules.google.com/task/7451371237623771014) started by @GarethClarridge*

### CODE REVIEW

File: tests/Unit/Http/Requests/UpdateMeetingRequestTest.php [hunk 1]
```diff
@@ -0,0 +1,342 @@
...
+        if (isset($data['_existing_meeting'])) {
+            $meeting = $data['_existing_meeting'];
+            unset($data['_existing_meeting']);
+        }
+
+        $this->request->setRouteResolver(function () use ($meeting) {
+            $route = $this->getMockBuilder(\Illuminate\Routing\Route::class)
+                ->disableOriginalConstructor()
+                ->getMock();
+            $route->method('parameter')->with('meeting')->willReturn($meeting);
```
[the remark this answers is not in the document]
Comment (GarethClarridge) [reply tests/Unit/Http/Requests/UpdateMeetingRequestTest.php]: @jules please fix this
